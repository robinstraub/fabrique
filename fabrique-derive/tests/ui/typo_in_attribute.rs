use fabrique_derive::Factory;

#[derive(Factory)]
struct Anvil {
    #[fabrique(primery_key)]
    id: u32,
    weight: u32,
}

fn main() {}