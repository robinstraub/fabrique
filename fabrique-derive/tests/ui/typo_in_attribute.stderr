error: Unknown field: `primery_key`. Did you mean `primary_key`?
 --> tests/ui/typo_in_attribute.rs:4:1
  |
4 | struct Anvil {
  | ^^^^^^
